chain-signatures/contract/src/config.rs

```diff
@@ -0,0 +1,30 @@
+use borsh::{self, BorshDeserialize, BorshSerialize};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Serialize, Deserialize, BorshSerialize, BorshDeserialize, PartialEq, Eq)]
+pub struct Config {
```

Comment:
We can add a config that @DavidM-D was talking about — parameters for a function that calculates the required deposit. Conversely, we can hardcode all the parameters and reduce complexity. Update the contract each time we need to change something. It is a bit more risky though.

Reply:
I'm planning on adding dynamic configs so we don't have to change the contract just to update the config structure so there's little risk here. Nodes can have their own version of these configs they can override as well